File: Ruby/test/example2.rb
# encoding: UTF-8
require 'ltpservice'

client = LTPService.new('username', 'password')

ltml = LTML.new
ltml.build_from_words(['我', '爱', '北京', '天安门'], 'UTF-8')
puts client.analyze(ltml)

File: Python/test/example2.py
# -*- coding:utf8 -*-
import ltpservice
from account import username, password

client = ltpservice.LTPService("%s:%s" % (username, password))
ltml = ltpservice.LTML()
ltml.build_from_words(["我", "爱", "北京", "天安门"])

result = client.analysis(ltml, ltpservice.LTPOption.PARSER)
print result.tostring()
